db/schema.rb

```diff
@@ -49,6 +49,14 @@
     t.index ["user_id"], name: "index_event_receivers_on_user_id", using: :btree
```

Comment:
Can we add to `event_receivers`:
- A string `type` not null for STI
- A text `regex` for output matching (text because we don't want to limit the size)
- A string `stream` for which stream to match the regex against
- An integer `return_code` for return code matching